tools/objinfo.ml

```diff
@@ -113,8 +113,8 @@ open Cmx_format
 let print_cmx_infos (ui, crc) =
   print_general_infos
     ui.ui_name crc ui.ui_defines ui.ui_imports_cmi ui.ui_imports_cmx;
-  printf "Approximation:\n";
```

Comment:
Not a big deal, but it would be useful to keep the ability to print (at least non-flambda) approximations here.